tests/pytests/unit/utils/test_stringutils.py

```diff
@@ -0,0 +1,584 @@
+"""
+Tests for stringutils utility file.
+"""
+
+import re
+import sys
+import textwrap
+
+import pytest
+import salt.utils.stringutils
+from salt.ext.six.moves import builtins, range  # pylint: disable=redefined-builtin
+from tests.support.mock import patch
+from tests.support.unit import LOREM_IPSUM
+
+UNICODE = "中国語 (繁体)"
```

Comment:
We want to try to avoid module level variables as these have caused headaches in the past when clashing with other test modules in the future.